projects/control-service/projects/helm_charts/pipelines-control-service/templates/alertmanager.yaml

```diff
@@ -0,0 +1,53 @@
+{{- /*
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.alertmanager.enabled }}
+apiVersion: v1
+kind: List
+metadata:
+  resourceVersion: ""
+items:
+  - apiVersion: monitoring.coreos.com/v1alpha1
+    kind: AlertmanagerConfig
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/last-applied-: ""
+      generation: 6
+      labels:
+        role: infra
+      name: dp-notifications
+      namespace: {{ .Values.alertmanager.items.namespace }}
+
+    spec:
+      receivers:
+        - emailConfigs:
+            - from: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.from }}
+              headers:
+                - key: subject
+                  value: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.headers.value }}
+              hello: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.hello }}
+              html: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.html }}
+              requireTLS: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.requireTLS }}
+              sendResolved: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.sendResolved }}
+              smarthost: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.smarthost }}
+              to: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.to }}
+              authUsername: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.authUsername }}
+              authPassword: {{ .Values.alertmanager.items.spec.receivers.emailConfigs.authPassword }}
```

Comment:
Storing sensitive information directly the yaml file is not a best practice for security reasons. It's better to use Kubernetes Secrets to manage sensitive information. The secret can be created separately and its name can be referenced in the values.yaml file 

Search for `kadmin-password` for a good example in our chart

I know there are some other passwords that are passed this way. But those are "grandfathered in" and we will change them at some point. I'd rather